Content.Server/Chemistry/EntitySystems/InjectorSystem.cs

```diff
@@ -114,6 +115,12 @@ private void OnInjectorAfterInteract(Entity<InjectorComponent> entity, ref After
     /// </summary>
     private void InjectDoAfter(Entity<InjectorComponent> injector, EntityUid target, EntityUid user)
     {
+        if (HasComp<BlockInjectionComponent>(target)) // DeltaV
+        {
+            Popup.PopupEntity(Loc.GetString("injector-component-deny-user"), target, user);
+            return;
+        }
```

Comment:
I strongly dislike that this is explicitly hardcoded only for Chitinids, rather than being a generic component using a localization datafield so that other things such as Hardsuits can define the message and conditions for how they block injections.